src/main/io/displayport_msp_bf_compat.c

```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 /*
     case SYM_HEADING:
         return BF_SYM_HEADING;
+*/
 
     case SYM_SCALE:
-        return BF_SYM_SCALE;
+        return 'I';
 
     case SYM_HDP_L:
-        return BF_SYM_HDP_L;
+        return BF_SYM_RSSI;
```

Comment:
I think using the RSSI symbol would be confusing. Maybe use 'H' for HDP_L and 'P' for HDP_R?